File: 3555-final-array-state-after-k-multiplication-operations-i/final-array-state-after-k-multiplication-operations-i.py
class Solution:
    def getFinalState(self, nums: List[int], k: int, multiplier: int) -> List[int]:
        n = len(nums)

        for i in range(k):
            minI = 0
            for j in range(n):
                if(nums[j]<nums[minI]):
                    minI = j
            
            nums[minI] *= multiplier

        return nums

File: 3555-final-array-state-after-k-multiplication-operations-i/final-array-state-after-k-multiplication-operations-i.cpp
class Solution {
public:
    vector<int> getFinalState(vector<int>& nums, int k, int multiplier) {
        int n = nums.size();

        for(int i=0; i<k; i++){
            int minI = 0;
            for(int j=0; j<n; j++){
                if(nums[j]<nums[minI]){
                    minI = j;
                }
            }
            nums[minI] *= multiplier; 
        }

        return nums;
    }
};